examples/walkthrough/README.md

```diff
@@ -0,0 +1,147 @@
+# Walkthrough
+
+## Setup
+
+### [Install Dependencies](https://github.com/smartcontractkit/chainlink#install)
+
+For this walkthrough, you will need:
+
+- [Go](https://golang.org/doc/install#install)
+- [dep](https://github.com/golang/dep#installation)
+- [Node](https://nodejs.org/en/download/package-manager/)
+- [Yarn](https://yarnpkg.com/lang/en/docs/install/#mac-stable)
+- [Docker](https://www.docker.com/get-started)
+- [direnv](https://direnv.net/) (optional)
+
+If you're on a mac with [Homebrew](https://brew.sh/), you can run:
+
+`brew install go dep node yarn docker direnv`
+
+### Install Chainlink
+
+
+```bash
+go get -d github.com/smartcontractkit/chainlink
+cd $GOPATH/src/github.com/smartcontractkit/chainlink
+make install
+chainlink help
+```
+
+## Run
+
+Spin up a local Ethereum node:
+
+```bash
+./internal/bin/devnet
+```
+
+In a separate window:
+
+```bash
+CHAINLINK_DEV=true chainlink node
+... login ...
+> [INFO]  Link Balance for 0x79dBA5B14cBA2560360c2eF48e9329aC7Ab21573
+```
+
+In a final window, which you'll use for the rest of the walktrhough, save the Chainlink node's Ethereum address:
+```bash
+ORACLE_NODE=0x79dBA5B14cBA2560360c2eF48e9329aC7Ab21573
+```
+
+## Set up your LINK and Oracle contracts
+In the same windoe where you set ORACLE_NODE, move to the solidity directory: `cd solidity/`
```

Comment:
Typo: `windoe`.

Also, I think `cd solidity` should also be on a newline in "\`\`\`bash\`\`\`" clause like the others to be more recognizable. I missed it when looking at the `./bin/fund_address`